### ISSUE TITLE
/Back into arenas

### ISSUE THREAD

**Issue by LogGits** [kangarko/CoreArena#145]
Players can /back into arenas after they are over. Can you add a way where they can't do this.

**Comment by LogGits** [comment]
People are still able to /back into arenas though? on the latest build of corearena and gameapi.

**Comment by kangarko** [comment]
In the next version ;)

**Comment by LogGits** [comment]
I just saw the update changelog. I am using [CMI](https://www.spigotmc.org/resources/cmi-240-commands-insane-kits-portals-essentials-economy-mysql-sqlite-much-more.3742/) (the better premium server content manager) to handle /back and tp's. Will this feature still work with cmi?

**Comment by kangarko** [comment]
Nope, that&#39;s for Essentials. I can look into CMI but their API seems more obscure, will do it in my free time.
